oci/tests/BUILD.bazel

```diff
@@ -29,10 +29,14 @@ diff_test(
     file1 = "java17",
     # This one is declared with an oci_pull rule in /WORKSPACE
     file2 = "@distroless_java",
+    # TODO: fix these tests.
```

Comment:
maybe file an issue at the same time, these are easy to forget

Reply:
done